### PR TITLE
Decorate fancy UI garbage

### PR DESCRIPTION
**Pull request by frostburn**
Add unique identifiers to garbage slabs for identity tracking.
Use svg.js to decorate the garbage slabs in fancy UI.

### CODE REVIEW

**Comment by frostburn** [comment]
svg_garbage

File: ui/fancy/src/js/actions.js [hunk 1]
```diff
@@ -63,10 +63,13 @@ module.exports = {
       const width = random(1, ui.game.width);
       const x = random(0, ui.game.width - width);
       const height = random(1, 3);
+      const uuid = Math.random();  // TODO: Actual UUID
```
Comment (RauliL) [inline ui/fancy/src/js/actions.js]: https://www.npmjs.com/package/uuid

File: ui/fancy/src/js/grid.js [hunk 2]
```diff
@@ -91,25 +92,55 @@ class Grid {
...
-  updateGarbage(state) {
-    // Variadic garbage slabs.
-    while (this.garbageElements.length < state.garbage.length) {
-      const newElement = document.createElement('div');
-      newElement.classList.add('garbage');
-      this.gridElement.appendChild(newElement);
-      this.garbageElements.push(newElement);
+  decorateGarbage(element, slab) {
+    const rand = Math.random;
+    const id = '' + rand();
```
Comment (RauliL) [inline ui/fancy/src/js/grid.js]: Use
```JavaScript
`${rand()}`
```
instead.

**Comment by RauliL** [comment]
Otherwise nice.

**Pull request merged**